tox.ini

```diff
@@ -9,27 +9,29 @@ skipsdist = True
 
 [testenv]
 extras = dev
+basepython = python3.10
+allowlist_externals = pdm
 
 [testenv:test]
-deps = -r operator-pipeline-images/requirements-dev.txt
-       -r operator-pipeline-images/requirements.txt
-
+groups = operator-pipelines
+         operator-pipelines-dev
+allowlist_externals = pytest
```

Comment:
This should not be here because you are not using the pytest from the pdm installation but instead using the installation from some other place..

Reply:
Updated!